source/game.c

```diff
@@ -254,14 +259,45 @@ bool is_shortcut_joker_active(void)
 }
 #define STRAIGHT_AND_FLUSH_SIZE_FOUR_FINGERS 4
 #define STRAIGHT_AND_FLUSH_SIZE_DEFAULT 5
-static int four_fingers_joker_count = 0;
+//static int four_fingers_joker_count = 0;
 static int straight_and_flush_size = STRAIGHT_AND_FLUSH_SIZE_DEFAULT;
 
 int get_straight_and_flush_size(void) 
 {
     return straight_and_flush_size;
 }
 
+static inline void _set_shop_joker_avail(int joker_id, bool avail)
```

Comment:
@MeirGavish Here are the interfaces to the bitmap we talked about. There is one use of a `BitmapItr` below, but it's in another `Joker` function, so I'm happy with it.